chia/util/config.py

```diff
@@ -162,3 +164,34 @@ def traverse_dict(d: Dict, key_path: str) -> Any:
         return val
     else:
         raise KeyError(f"value not found for key: {key}")
+
+
+start_methods: Dict[Optional[str], Optional[str]] = {
+    None: None,
+    "default": None,
+    "fork": "fork",
+    "forkserver": "forkserver",
+    "spawn": "spawn",
+}
+
+
+def process_config_start_method(
+    method: object,
+    log=logging.Logger,
+) -> Union[None, Literal["fork"], Literal["fork_server"], Literal["spawn"]]:
+    sentinel = object()
+    # TODO: I am aware that some objects may not be in the dict, that is why I am
+    #       providing .get() with a default.
+    processed_method = start_methods.get(method, sentinel)  # type: ignore[call-overload]
+
+    if processed_method is sentinel:
```

Comment:
Why not just use None? it would make the code more readable.

Reply:
Because `None` is a valid result and thus not distinguishable from invalid input.